extension/manifest.json

```diff
@@ -1,32 +1,40 @@
 {
   "manifest_version": 2,
-  "name": "speak-to-me",
+  "name": "Voice Fill",
   "version": "1.2.1",
+  "description": "Adds voice input to popular search pages in Firefox. Learn more about Voice Fill at https://testpilot.firefox.com",
   "developer": {
     "name": "Fabrice Desré",
```

Comment:
I discussed with Faramarz and he said we should use: `Emerging Technologies Advanced Dev Team`, but I also agree we should list all individuals who worked in the project in the experiment page.